examples/To_pardiso/test_pardiso_new.py

```
@@ -0,0 +1,103 @@
+    
+    cmd = [
+        "julia",
+        julia_script,
+        "--input_dir", output_dir_new,
+        "--output_dir", julia_out_dir,
+        "--config", config_path
+    ]
+    
+    print(f"Running: {' '.join(cmd)}")
+    try:
+        subprocess.run(cmd, check=True)
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)